src/app/app.module.ts

```diff
@@ -1,16 +1,38 @@
 import { BrowserModule } from '@angular/platform-browser';
 import { NgModule } from '@angular/core';
+import { RouterModule } from '@angular/router';
```

Comment:
is there a reason to use js for backend while ts for frontend? one of the biggest advantage of node is there isn't any language context-switch between frontend & backend coding.